src/main/scala/scalaz/parsers/tc/Alternative.scala

```diff
@@ -27,9 +26,6 @@ object Alternative {
   implicit def eitherTAlternative[F[_]: Monad, L]: Alternative[EitherT[L, F, ?]] =
     new Alternative[EitherT[L, F, ?]] {
       override def or[A](f1: EitherT[L, F, A], f2: => EitherT[L, F, A]): EitherT[L, F, A] =
-        EitherT(f1.run.flatMap {
-          case -\/(_) => f2.run
-          case \/-(_) => f1.run
-        })
+        f1.orElse(f2)
```

Comment:
Is this where the singularity happened?

Reply:
Yes. `run` was responsible for running 😜